resolver/sudn_resolver.go

```diff
@@ -0,0 +1,145 @@
+package resolver
+
+import (
+	"fmt"
+	"net"
+	"strings"
+
+	"github.com/0xERR0R/blocky/model"
+	"github.com/miekg/dns"
+)
+
+const (
+	sudnTest      = "test."
+	sudnInvalid   = "invalid."
+	sudnLocalhost = "localhost."
+)
+
+func sudnArpaSlice() []string {
+	return []string{
+		"10.in-addr.arpa.",
+		"21.172.in-addr.arpa.",
+		"26.172.in-addr.arpa.",
+		"16.172.in-addr.arpa.",
+		"22.172.in-addr.arpa.",
+		"27.172.in-addr.arpa.",
+		"17.172.in-addr.arpa.",
+		"30.172.in-addr.arpa.",
+		"28.172.in-addr.arpa.",
+		"18.172.in-addr.arpa.",
+		"23.172.in-addr.arpa.",
+		"29.172.in-addr.arpa.",
+		"19.172.in-addr.arpa.",
+		"24.172.in-addr.arpa.",
+		"31.172.in-addr.arpa.",
+		"20.172.in-addr.arpa.",
+		"25.172.in-addr.arpa.",
+		"168.192.in-addr.arpa.",
+	}
+}
```

Comment:
Maybe this can be a `const` instead of a function?

Reply:
Thought it would be more readable using a slice instead of a bunch const strings.
As slices can't be const best practice is to create a function returning a fixed slice (as far as i know).
Please correct me if I'm wrong. 😅

Reply:
Ah didn't remember that of the top of my head :laughing: 

Fine to ignore then!